centreon/www/include/views/graphTemplates/DB-Func.php

```
@@ -75,31 +72,41 @@ function deleteGraphTemplateInDB($graphs = []): void
+        'lower_limit', 'upper_limit', 'size_to_max', 'default_tpl1',
+        'scaled', 'stacked', 'comment', 'split_component',
+    ];
+    $selectStmt = $pearDB->prepare(
+        'SELECT ' . implode(', ', $columns) . ' FROM giv_graphs_template WHERE graph_id = :graphTemplateId LIMIT 1'
+    );
+    $placeholders = implode(', ', array_map(fn ($col) => ':' . $col, $columns));
+    $insertStmt = $pearDB->prepare(
+        'INSERT INTO giv_graphs_template (' . implode(', ', $columns) . ') VALUES (' . $placeholders . ')'
+    );
+
+    foreach ($graphs as $key => $value) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)